packages/core/src/types/Vector.ts

```diff
@@ -117,10 +117,39 @@ export class Vector2 implements Type {
     return new Vector2(Math.cos(radians), Math.sin(radians));
   }
 
+  /**
+   * Return the angle in radians between the vector described by x and y and the
+   * positive x-axis.
+   *
+   * @param x - The x component of the vector.
+   * @param y - The y component of the vector.
+   */
+  public static radians(x: number, y: number) {
+    return Math.atan2(y, x);
+  }
+
+  /**
+   * Return the angle in degrees between the vector described by x and y and the
+   * positive x-axis.
+   *
+   * @param x - The x component of the vector.
+   * @param y - The y component of the vector.
+   *
+   * @remarks
+   * The returned angle will be between 0 and 360 degrees.
```

Comment:
Idk about this chief, `Vector2.radians()` returns the range `[-PI, PI]`
Shouldn't this return `[-180, 180]` too?

Reply:
I don't feel super strongly about this since this wouldn't make any difference when using it for rotating a node. I just found `[0, 360]` a bit more intuitive but I can change it if you'd prefer it to be consistent with `radians`

Reply:
I'd go for consistency